biz.aQute.junit.platform.reporting/bnd.bnd

```diff
@@ -0,0 +1,8 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+
+-includeresource.reporting: @${repo;org.junit.platform:junit-platform-reporting;${junit-platform-reporting}}
```

Comment:
We should use `latest` for the version and control that via central.mvn.

Reply:
Done.